### DISCUSSION TITLE
January 2021: OpenSMTPD libtls conversion and UNIX-domain sockets support, nooSMTPD · poolp.org

### DISCUSSION THREAD

**Discussion by poolpOrg** [poolpOrg/poolp.org#126] [Articles]

**Comment by ArchangeGabriel** [comment]
Awesome, looking forward to see the changes on the SSL side (especially per listener tuning) and what nooSMTPD becomes (for what I’ve read currently, this looks interesting to me).

**Comment by poolpOrg** [comment]
you should read the February report then ;)
